main/settings_celery.py

```diff
@@ -131,12 +131,6 @@
         "schedule": crontab(minute=30, hour=18),  # 2:30pm EST
         "kwargs": {"period": "daily", "subscription_type": "channel_subscription_type"},
     },
-    "daily_embed_new_learning_resources": {
```

Comment:
@rhysyngsun this task was intentionally removed [here](https://github.com/mitodl/mit-learn/pull/2001) so that it only runs in a live environment but was re-instated [here](https://github.com/mitodl/mit-learn/commit/73ec984378ea914cfe85ac510562d891966cc53b#diff-35a3042773b7652baf679eb2fe743e78debcf410d4ed6c72196fac1d9f52d8b8R134) - im re-removing it again in this pr so just want to make sure i'm not missing anything